third-party/glad/glad.c

```diff
@@ -855,6 +855,8 @@ PFNGLGETOBJECTLABELKHRPROC glad_glGetObjectLabelKHR = NULL;
 PFNGLOBJECTPTRLABELKHRPROC glad_glObjectPtrLabelKHR = NULL;
 PFNGLGETOBJECTPTRLABELKHRPROC glad_glGetObjectPtrLabelKHR = NULL;
 PFNGLGETPOINTERVKHRPROC glad_glGetPointervKHR = NULL;
+#pragma GCC diagnostic push
```

Comment:
1. This is a great initiative
2. This warning is on Linux only - lets wrap the lines with `#if defined __linux__ && #endif`
3. Lets also add a notice that we manually change this auto generated file to avoid compiler warnings